src/components/filter.js

```diff
@@ -81,16 +89,24 @@ const Filter = ({
       <ListItem
         innerDivStyle={style.listItemInner}
         leftIcon={<SettingsIcon style={style.listIcon} />}
-        primaryText={<FormattedMessage {...messages.changeFilters} />}
-      />
+      >
+        <Link
```

Comment:
`Link` creates a `<a>` behind the scenes with a real, live href (no JS). This means that no `onClick` handler needs to be passed in as props.

CSS tweaked in c37f181.